docs/platform/advanced-targeting/custom-properties.md

```diff
@@ -144,3 +144,44 @@ You can target users in numerous ways using Custom Properties. The following lis
 **Special Users**. Sometimes organizations want to release a feature to special users only, such as users with a paid membership, those with a free trial, or those who have made large contributions to the company. In this case, some property name suggestions are `accountType` `pricingPlan` `isSubscriber` `isTrialUser` `amountContributed` etc.
 
 **User Behaviour or Preferences.** You can experiment on users based on their behaviour or preferences. Some examples include `numberOfPageVisits` `gaveConsent` `preferredColor` etc.
+
+## Custom Property Schemas
+
+Similar to [Variable Schemas](/platform/advanced-variables/variable-schemas.md), Custom Property Schemas allow you to set up a schema and define a list of possible values for a Custom Property, which will appear in a dropdown when setting up Targeting Rules. Labels can be defined for each value to make selections more human-readable and easier to identify. This removes the need to remember valid inputs for a specific custom property.
+
+At DevCycle, we pass each organization's ID as a Custom Property that we use to turn on Features for specific customers. Schemas allow us to set up allowable organization ID values, along with their human-readable names, which makes it easy to set up Targeting Rules without needing to double-check for a correct organization ID.
+
+![Custom Property Schema Targeting Rule](/oct-2024-cust-prop-schema-targ-rule.png)
+
+### Setting Up a Custom Property Schema 
+
+You can find a list of all existing Custom Properties in the Project Settings section of your [Settings page](https://app.devcycle.com/r/settings). 
+
+![Custom Property Section on Settings Page](/oct-2024-cust-prop-settings-page.png)
+
+To add a schema to a Custom Property, click on the Custom Property you want to modify. On the Custom Property Details page, locate the Schema section. Schemas can be added to Custom Properties that are already in use.
+
+![Schema Section on Details Page](/oct-2024-schema-on-cust-prop-det-page.png)
+
+From there, select an Enum Schema from the dropdown. This allows you to specify allowable values for string or number properties.
+
+If you want to require this property to be set when passing custom data to the SDK, click the toggle icon next to Property Required. 
+
+Enabling Property Required provides an additional safeguard when passing user data into the SDK. When enabled, TypeScript enforces that a specified property, such as org_id, is present in the custom data. If any custom property flagged as required is missing, the SDK will generate a TypeScript error during development, alerting you to missing data essential for your targeting rules to work correctly in the dashboard.
```

Comment:
i would move all this stuff inside the notice about how it affects type generators